qml/keys/languages.js

```diff
@@ -32,6 +32,7 @@ function languageIdToName(languageId)
     if (languageId == "fa")         return i18n.tr("Persian");
     if (languageId == "fi")         return i18n.tr("Finnish");
     if (languageId == "fr")         return i18n.tr("French");
+    if (languageId == "fr-ch")      return i18n.tr("French\n(Swiss)");
```

Comment:
Because it belongs on a new line? Why should there not be a new one? I'd argue these should all have {} as well, but I'm not going to make Dalton fix that in this PR.